libr/core/cmd_print.c

```diff
@@ -1361,7 +1361,8 @@ static void cmd_print_format(RCore *core, const char *_input, const ut8* block,
 		while (*fmt && IS_WHITECHAR (*fmt)) {
 			fmt++;
 		}
-		int struct_sz = r_print_format_struct_size (fmt, core->print, mode, 0) + 10;
+		int times = atoi (fmt);
```

Comment:
i think this "times" should be should be computed inside the r_print_format_struct_size, and i think this change s not solvinig the problem, (let me explain below)

Reply:
oh wait, i think you are right, but the struct size should be computed inside the r_print_format_struct_size function, because the bug only affects when using a repeater